### PR TITLE
add ownership slides

### PR DESCRIPTION
**Pull request by malcolmbarrett**

### CODE REVIEW

**Review by jonthegeek — DISMISSED** [review]
Thanks!

File: slides/04-understanding_ownership.qmd [hunk 1]
```diff
@@ -2,30 +2,204 @@
...
+println!("{x}");
+println!("{x}");
+```
+
+## Stack
+
+```{r}
+#| eval: true
+#| echo: false
+knitr::include_graphics("~/core/active/reference/bookclub-rust/slides/img/stack.png")
```
Comment (DrEntropy) [inline slides/04-understanding_ownership.qmd:68]: This probably should be just "img/stack.png".   This is why the build is failing I believe.

Comment (jonthegeek) [reply slides/04-understanding_ownership.qmd:68]: If you put this in as a code suggestion, I can accept it on my phone.

Comment (DrEntropy) [inline slides/04-understanding_ownership.qmd:68]:
```suggestion
knitr::include_graphics("img/stack.png")
```

Comment (DrEntropy) [reply slides/04-understanding_ownership.qmd:68]:
```suggestion
knitr::include_graphics("img/stack.png")
```

**Review by jonthegeek — APPROVED** [review]
Looks like that fixed it!

**Pull request merged**